integration/system/disk_usage_test.go

```diff
@@ -53,20 +54,18 @@ func TestDiskUsage(t *testing.T) {
 				assert.NilError(t, err)
 				assert.Assert(t, du.LayersSize > 0)
 				assert.Equal(t, len(du.Images), 1)
-				assert.DeepEqual(t, du, types.DiskUsage{
-					LayersSize: du.LayersSize,
-					Images: []*types.ImageSummary{
-						{
-							Created:  du.Images[0].Created,
-							ID:       du.Images[0].ID,
-							RepoTags: []string{"busybox:latest"},
-							Size:     du.LayersSize,
-						},
-					},
-					Containers: []*types.Container{},
-					Volumes:    []*volume.Volume{},
-					BuildCache: []*types.BuildCache{},
-				})
+				assert.Equal(t, len(du.Images[0].RepoTags), 1)
+				assert.Equal(t, du.Images[0].RepoTags[0], "busybox:latest")
+
+				//Image size is layer size + content size, should be greater than total layer size
+				assert.Assert(t, du.Images[0].Size >= du.LayersSize)
+
+				// If size is greater, than content exists and should have a repodigest
+				if du.Images[0].Size > du.LayersSize {
+					assert.Equal(t, len(du.Images[0].RepoDigests), 1)
+					assert.Assert(t, strings.HasPrefix(du.Images[0].RepoDigests[0], "busybox@"))
```

Comment:
If an assertion failure is "safe" (won't cause a panic later), it's better to use `assert.Check` instead of `assert.Assert` as it won't abort immediately and will continue with other asserts.